flashing/flash_firmware.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+
+from pathlib import Path
+
+from pyocd.core.helpers import ConnectHelper
```

Comment:
It would be great to pull these dependencies into a requirements.txt.

Reply:
Ok, it is basically only `pyocd`. The remaining dependencies are either transient or stdlib. I created a `requirements.txt` for others to use.